### PR TITLE
removed scraping

### PR DESCRIPTION
**Pull request by piyx**
Getting image links directly without scraping.
It runs much faster now since processing of post links to get image link is not required.

### CODE REVIEW

File: add_images.py [hunk 1]
```diff
@@ -29,29 +29,12 @@
...
-
-    for link in soup.findAll("a"):
-        if (
-            "https://i.redd.it/" in str(link.get("href"))
-            or "external-preview.redd.it/" in str(link.get("href"))
-        ) and str(link.get("href")) not in images:
-            images.append(str(link.get("href")))
+for submission in subreddit.hot(limit=NUM_IMAGES):
+    img_url = submission.url
+    if os.path.splitext()[-1]:
```
Comment (Destaq) [inline add_images.py:36]: Could you please add `img_url` here or remove this if statement entirely so the code runs?

**Review by Destaq — CHANGES_REQUESTED** [review]
Please ensure `splitext` runs properly, or better yet, remove this conditional entirely.

**Comment by Destaq** [comment]
We also need to check for invalid links. Adding code like this (for example) would ensure the code runs error-free:
```python
    except FileNotFoundError:
        print("\n", fname, "has invalid link")
```

**Comment by piyx** [comment]
Alright I will remove the conditional and add Exceptions.

**Pull request closed without merging**